fractions/keycloak-server/src/main/resources/modules/org/keycloak/keycloak-themes/main/module.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8" standalone="yes"?>
-<module xmlns="urn:jboss:module:1.5" name="org.keycloak.keycloak-themes" slot="main">
+<module xmlns="urn:jboss:module:1.5" name="org.keycloak.keycloak-themes">
   <resources>
-    <resource-root path="."/>
+    <artifact name="org.keycloak:keycloak-themes:3.4.3.Final"/>
```

Comment:
Should that refer to a Maven property for the version instead of being hard coded?